bitnami/thanos/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/thanos
   - https://thanos.io
-version: 11.6.1
+version: 11.6.2
```

Comment:
Version `11.6.2` has been already released, could you please bump the chart version again? https://github.com/bitnami/charts/blob/main/bitnami/thanos/Chart.yaml